recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: behst
+  version: 29efb4c3a8e333a2824430ef32d8bf01
```

Comment:
Is there any real version number available? I fear that this will confuse conda in any future version if you change the hash. E.g. the hash could be "smaller" than an older one.